content/browser/webui/brave_shared_resources_data_source.cc

```diff
@@ -162,7 +167,12 @@ BraveSharedResourcesDataSource::GetAccessControlAllowOriginForOrigin(
   // back.
   std::string allowed_origin_prefix = content::kChromeUIScheme;
   allowed_origin_prefix += "://";
+  std::string allowed_untrusted_origin_prefix =
+      content::kChromeUIUntrustedScheme;
+  allowed_untrusted_origin_prefix += "://";
   if (!base::StartsWith(origin, allowed_origin_prefix,
+                        base::CompareCase::SENSITIVE) &&
+      !base::StartsWith(origin, allowed_untrusted_origin_prefix,
```

Comment:
the comment in https://github.com/brave/brave-core/pull/12906/files#diff-42fad6b1a88feeba63d2a1a4ba92afa8a20c0b78dbb96f3cef2fbfaeb21b7776R164 should be updated